tests/zeromq.terminal/PubSubTests/PubSubScenarios.cs

```diff
@@ -47,7 +48,7 @@ internal static async Task PubSubWithCancellation(SocketConfiguration configurat
             {
                 await socket.PublishAsync(new PubSubMessage { Message = "Published with cancellation"});
             }
-            System.Console.WriteLine("press any key to exit");
+            System.Console.WriteLine("press enter to exit");
```

Comment:
We don't check that the `Enter` is pressed, but just any key is fine. Should we still change the text from "any key" to "enter"?

Reply:
it is Readline (see other comment)

Reply:
`Console.ReadLine` ... my bad